doc/content/Administration/config.md

```diff
@@ -1,4 +1,8 @@
-# Configuration
+---
```

Comment:
Not convinced by calling this "Administration". This is about **operating** a server, that's not the same.

Reply:
Me neither at first I named "Operational" but eventually I wanted to but put instruction on how to *administer* or manage the stack much like https://docs.influxdata.com/influxdb/v1.7/administration/ .